lib/util/generate-subcodes-doc.js

```diff
@@ -0,0 +1,72 @@
+const path = '../config/error-codes/';
+const internal = require(`${path}internal`);
+const external = require(`${path}external`);
+const api = require(`${path}/api`);
+const network = require(`${path}network`);
+const plugins = require(`${path}plugins`);
+const fs = require('fs');
+
+function errorLink(errorClass) {
+  switch (errorClass) {
+    case 'BadRequestError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'ForbiddenError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'KuzzleError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'ParseError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'PluginImplementationError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'SizeLimitError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'ExternalServiceError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'GatewayTimeoutError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'InternalError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'NotFoundError':
+      return `[${errorClass}](https://docs.kuzzle.io/core/1/api/essentials/errors/#${errorClass.toLowerCase()})`;
+    case 'PartialError':
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Either merge this case with the identical one on line "29" or change one of the implementations. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1871)